internal/pkg/term/selector/selector.go

```diff
@@ -38,9 +38,10 @@ type ConfigLister interface {
 	ConfigSvcLister
 }
 
-// WsSvcLister wraps the method to get svcs in current workspace.
-type WsSvcLister interface {
+// WsWlLister wraps the method to get workloads in current workspace.
+type WsWlLister interface {
```

Comment:
nit: I find this a bit hard to read, should we just spell it out? 
```suggestion
type WorkloadsLister interface {
```

Reply:
How about WsWorkloadLister?